__tests__/articles.test.js

```diff
@@ -131,4 +131,40 @@ describe("/api/articles", () => {
             expect(idOrder).toEqual([3, 6,  2, 12, 5, 1, 9, 10, 4, 8, 11, 7])
         })
     })
+})
+
+describe("/api/articles/:article_id/comments", () => {
+    describe("GET", () => {
+        test("Status 200 - returns the correct number of comment objects", async () => {
+            const output = await request(app).get('/api/articles/1/comments')
+            expect(output.status).toBe(200)
+            expect(output.body.comments.length).toBe(11)
+        })
+        test("Status 200 - returned comment objects contain the expected keys", async () => {
+            const output = await request(app).get('/api/articles/1/comments')
+            const comments = output.body.comments
+            comments.forEach((comment) => {
+                expect(comment.hasOwnProperty("comment_id")).toBe(true)
+                expect(comment.hasOwnProperty("votes")).toBe(true)
+                expect(comment.hasOwnProperty("created_at")).toBe(true)
+                expect(comment.hasOwnProperty("author")).toBe(true)
+                expect(comment.hasOwnProperty("body")).toBe(true)
+            })
+        })
+        test("Status 200 - returns an empty array when the article exists but there are no comments", async () => {
```

Comment:
perhaps worth testing whether the status code is what you expect as part of this test